core-domain/src/main/java/org/uniprot/core/uniprotkb/impl/UniProtKBEntryImpl.java

```diff
@@ -562,7 +562,7 @@ private void updateProteinSectionEvidences(Set<Evidence> evidences, ProteinSecti
         }
     }
 
-    private void updateProteinRecNameEvidences(Set<Evidence> evidences, ProteinRecName pn) {
+    private void updateProteinRecNameEvidences(Set<Evidence> evidences, ProteinName pn) {
```

Comment:
I think updateProteinRecNameEvidences and updateProteinAltNameEvidences might be merged, right?

Reply:
ah good spot, done